nixio/file.py

```diff
@@ -200,6 +202,22 @@ def format(self):
     def format(self, f):
         util.check_attr_type(f, str)
         self._root.set_attr("format", f.encode("ascii"))
+        if self.time_auto_update:
+            self.force_updated_at()
+
+    @property
+    def time_auto_update(self):
+        """
+        A user defined flag which decided if time should always be updated
+        when properties are changed.
+
+        :type: bool
+        """
+        return self._time_auto_update
+
+    @time_auto_update.setter
+    def time_auto_update(self, bool):
```

Comment:
Can you change the name of the parameter here?  `bool` is the buit-in name for the bool type.

Reply:
changed